docs/preset-flow.md

```diff
@@ -91,4 +91,17 @@ class A {
 }
 ```
 
+:::babel8
+
+### `ignoreExtensions`
```

Comment:
Don't we support this option in Babel 7?

Reply:
Indeed. Looking back that PR should have been marked as New Features so we wouldn't have shipped it in a patch release.